submissions/career/career.md

```diff
@@ -0,0 +1,53 @@
+
+Several months later, my cousin came to visit and saw I had Factorio - he was excited to play it so we decided that together we would give it a go. After some sketchy library sharing (yay Steam offline mode), we spent a weekend of bliss together, figuring out how the game worked and progressing past the ever challenging blue science packs. *Side note: This is a really great example of Piracy for good since that weekend of Factorio caused my cousin to go home and buy the game himself so we could keep playing together, something that might not have happened if not for our dodgy dealings.* This push past the percieved complexity forever ruined me, it caused the crippling Cracktorio addiction that many of us share, meaning I never could put this beast back down.
+
+So... why am I telling you this? *Fantastic question, really glad you asked. Very well asked question I might add.* For me it's pretty clear that playing modded Minecraft and Factorio pushed forward my critical thinking and mathematical skills and diverted me down the STEM path of goodness. Now, I'm studying Robotic Engineering and though Factorio isn't the only thing that pushed me down this path, it definitely helped guide me to where I am now.
+
+### The computer science angle
+
+I'm not alone in the experience of Factorio guiding my choices elsewhere in life. So many people I've talked to, including our very own *Therenas - ~~LORD GOD~~ Editor-in-chief of Alt-F4*, have starting playing Factorio, experimented with mods then been forced into learning to program so they can fufill their desire to create their own mods. I would *blindly, and with no evidence* suggest there are a not insignificant number of people who learned to program because of Factorio and other moddable games. Even just playing around with combinators and learning the basic of computer logic provides a new skill for many people, maybe pushing them slightly closer to the ever loved STEM fields.
+
+![Playing to modding Factorio meme](https://cdn.discordapp.com/attachments/603392474458882065/745728165116248144/mod_brain.png)
+*Source: Find creator*
+
+### Correlation vs Causation
+
+In writing this piece,  I made a lovely little [post](https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/) on the r/Factorio subreddit, asking people about their relationship with Factorio and STEM, and how it has inspired/changed their education and careers. As always with these kinds of things, I got some of what I wanted, and some interesting new perspectives I hadn't really considered.
```

Comment:
This one is also really nice, and I think might be worth an add
https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxj9g1x?utm_source=share&utm_medium=web2x&context=3